packages/database/src/migrations/20210922211939-LESMIS-number-of-schools-overlays-modifies-data.js

```diff
@@ -0,0 +1,324 @@
+'use strict';
+
+import { generateId, insertObject, codeToId, nameToId, arrayToDbString } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const getReport = (reportCode, dataElements) => {
+  const aggregations = [
+    {
+      type: 'FINAL_EACH_YEAR',
+      config: {
+        dataSourceEntityType: 'sub_district',
+        aggregationEntityType: 'requested',
+      },
+    },
+  ];
+
+  if (reportCode.includes('Province')) {
+    aggregations.push({
+      type: 'SUM_PER_ORG_GROUP',
+      config: {
+        dataSourceEntityType: 'sub_district',
+        aggregationEntityType: 'district',
+      },
+    });
```

Comment:
@reviewer I'm not too familiar with `SUM_PER_ORG_GROUP`, I assume this is what I want to do to see the sub_district data at province level?!

Reply:
Yup this is the correct usage of `SUM_PER_ORG_GROUP`

Also seem to be accidentally tagging a random github user

Reply:
Haha woops. I saw someone else do that and I thought it was a thing